app.js

```diff
@@ -1,13 +1,21 @@
 const express = require("express");
 const { getCategories } = require("./controllers/categories.controller.js");
+const getReviewById = require("./controllers/getReviewsById.controller")
 const { handle404endPoint } = require("./controllers/errorHandlers.controller");
+const { handlePSQL400s, handleCustomErrors, handle500Statuses } = require("./controllers/errorControllers.controllers");
 
 const app = express();
 
 app.use(express.json());
```

Comment:
This shouldn't be here - please delete and make sure it doesn't appear again until it is needed (patch or post)

Reply:
Yep got it, many thanks :)